packages/Checkbox/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "LeafyGreen UI Kit Checkbox",
   "main": "./dist/index.js",
   "scripts": {
-    "build": "../../node_modules/.bin/webpack --config ../../webpack.config.js"
+    "build": "../../node_modules/.bin/webpack --config ../../webpack.config.js",
+    "ts:emit": "tsc --project ./tsconfig.json"
```

Comment:
Drive-by fix adding a line that I missed in #82

Reply:
Also... Is there some way we could lint for this?

Reply:
Would be neat to make an eslint rule for this. Or just a custom script to validate that we have this field present in a package.json file. But yeah agree, annoying we have to manually catch this.

Reply:
Even better – re-explore removing these redundant scripts. They're used to work around lerna, but it would be great to revisit this to remove the `build`/`ts:emit` scripts, as well as the package-specific `tsconfig.json`.